bootstrap-salt.sh

```diff
@@ -2663,6 +2669,11 @@ install_debian_deps() {
         __PIP_PACKAGES="${__PIP_PACKAGES} 'requests>=$_PY_REQUESTS_MIN_VERSION'"
     fi
 
+    if [ "$DISTRO_MAJOR_VERSION" -gt 8 ]; then
```

Comment:
@jbfriedrich Just give a try to the latest stable bootstrap version on Debian, git installation should work fine (on `amd64` arch).
But definitely, Ubuntu need to be fixed, so your related change looks good to me.